.quickstart/quickstart.yml

```diff
@@ -36,4 +36,4 @@ public_models: [
   "shopify__customers",
   "shopify__order_lines",
   "shopify__line_item_enhanced"
-]
+]
```

Comment:
Even though we won't add the new models just yet, we should add the new variables.